src/Main/TextUserInterface.cpp

```diff
@@ -1619,7 +1641,7 @@ namespace VeraCrypt
 						backupFile.ReadAt (headerBuffer, layout->GetType() == VolumeType::Hidden ? layout->GetHeaderSize() : 0);
 
 						// Decrypt header
-						shared_ptr <VolumePassword> passwordKey = Keyfile::ApplyListToPassword (options.Keyfiles, options.Password);
+						shared_ptr <VolumePassword> passwordKey = Keyfile::ApplyListToPassword (options.Keyfiles, options.Password, true);
```

Comment:
Same as above: EMVOption parameter is always set to true regardless of the actual EMV option value.

Reply:
Same as above : VC in CLI mode always checks for EMV cards